Archipelago.HollowKnight/Archipelago.cs

```diff
@@ -234,13 +235,25 @@ public void StartOrResumeGame(bool randomize)
             {
                 LogDebug("StartOrResumeGame: Beginning first time randomization.");
                 ApSettings.ItemIndex = 0;
+                ApSettings.Seed = Convert.ToInt32((long)loginResult.SlotData["seed"]);
```

Comment:
I'm not sure there's a reason to cast to `long` then `Convert.ToInt32()`. Couldn't you cast to an `int` and be done with it?

Reply:
... because I lifted the code from somewhere else that does that, presumably because of being used to seed a RNG.

Changed to longs throughout.